logsrvd/logsrvd.c

```diff
@@ -1172,16 +1279,23 @@ connection_closure_alloc(int sock)
     if (closure->commit_ev == NULL)
 	goto bad;
 
-    closure->read_ev = sudo_ev_alloc(sock, SUDO_EV_READ|SUDO_EV_PERSIST,
+    closure->read_ev = sudo_ev_alloc(sock, SUDO_EV_READ|SUDO_EV_PERSIST|SUDO_EV_TIMEOUT,
```

Comment:
You don't need to specify SUDO_EV_TIMEOUT here, calling sudo_ev_add() with a timeout pointer is sufficient.

Reply:
fixed and amended to the corresponding commit